source/Octopus.Tentacle/Configuration/WindowsMachineKeyEncryptor.cs

```diff
@@ -7,10 +7,12 @@ namespace Octopus.Tentacle.Configuration
 {
     public class WindowsMachineKeyEncryptor : IMachineKeyEncryptor
     {
+#pragma warning disable CA1416
```

Comment:
Just wondering, why did this become something we need to disable?

Reply:
It's due to a nuget bump, they have decorated methods that are OS-specific more widely and result in this warning. We only call this code when it's Windows so it's not of any concern (caller checks the OS first)

It could be cleaned up/made safer so it doesn't have this code path when we compile for anything other than Windows but didn't want to do that in this PR.